src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/AcasCertificateController.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.et.common.model.ccd.CCDCallbackResponse;
+import uk.gov.hmcts.et.common.model.ccd.CCDRequest;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.AcasService;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.VerifyTokenService;
+
+import java.util.List;
+
+import static org.springframework.http.HttpStatus.FORBIDDEN;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.CallbackRespHelper.getCallbackRespEntityErrors;
+
+@RestController
+@RequestMapping("/acasCertificate")
+@Slf4j
+public class AcasCertificateController {
+    private static final String INVALID_TOKEN = "Invalid Token {}";
+    private static final String GENERATED_DOCUMENT_URL = "Please download the ACAS Certificate from : ";
+    private final VerifyTokenService verifyTokenService;
+    private final AcasService acasService;
+
+    public AcasCertificateController(VerifyTokenService verifyTokenService, AcasService acasService) {
+        this.verifyTokenService = verifyTokenService;
+        this.acasService = acasService;
+    }
+
+    @PostMapping(value = "/retrieveCertificate", consumes = APPLICATION_JSON_VALUE)
```

Comment:
May be add log when this callback is called? I know we can argue against it as well but it is helpful in future investigating any issues we may have.

Reply:
Log messages for callbacks added